tests/Stripe/ApiRequestorTest.php

```diff
@@ -1,6 +1,6 @@
 <?php
 
-class Stripe_ApiRequestorTest extends UnitTestCase
+class Stripe_ApiRequestorTest extends PHPUnit_Framework_TestCase
```

Comment:
These tests did not use the original `StripeTestCase` As they did not need any of the additional functionality, i felt it was best to not add that here